worker-queue-rabbit/src/main/java/com/github/workerframework/queues/rabbit/WorkerQueueConsumerImpl.java

```diff
@@ -246,33 +265,34 @@ public void processAck(long tag)
             metrics.incremementErrors();
             consumerEventQueue.add(new ConsumerAckEvent(tag));
         }
+
```

Comment:
@andyreidz If the ack fails above, should we be falling through here and deleting the message from the datastore?